src/arcade/potts/env/location/PottsLocation.java

```diff
@@ -627,141 +631,36 @@ Location separateVoxels(ArrayList<Voxel> voxelsA, ArrayList<Voxel> voxelsB,
         calculateCenter();
         return makeLocation(voxelsB);
     }
-    
+
     /**
-     * Splits the voxels in the location along a given direction.
-     *
-     * @param direction  the direction of the slice
-     * @param voxels  the list of voxels
-     * @param voxelsA  the container list for the first half of the split
-     * @param voxelsB  the container list for the second half of the split
-     * @param center  the center voxel
-     * @param random  the seeded random number generator
-     */
-    static void splitVoxels(Direction direction, ArrayList<Voxel> voxels,
+     * Divides the voxels in the location into two lists along a given plane.
+     * <p>
+     * The voxels are split into two lists based on their position relative to
+     * the plane. Voxels on the plane are randomly assigned to one of the lists.
+     *
+     * @param plane the plane to split the voxels along
+     * @param voxels the list of voxels to split
+     * @param voxelsA the container list for the first half of the split.
```

Comment:
When descriptions of parameters that longer than a single sentence, put it in the body of the javadoc instead or simplify, e.g. `the container list for voxels on side of plane opposite the normal`